src/app/containers/CpsMetadata/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Metadata from '../Metadata';
+import { getBrandedImage } from './utils';
+
+const CpsMetadata = ({
```

Comment:
I am not so keen on including CPS in the name of the component...  Can we just add this functionality to the existing MetadataContainer, so that all components / page types can benefit from it, or does it have to be a separate component?

Reply:
It needs to be a separate component because we have different requirements from the articles page in regards to metadata. https://github.com/bbc/simorgh/blob/latest/src/app/containers/ArticleMetadata/index.jsx is the articles metadata which has some similarities but is quite different from this.

Reply:
Ah OK, makes sense.

Reply:
Maybe MediaMetadata?

Reply:
Media would imply only MAPs, but I think it is the same for STY & PGL... Let's leave it as CPSMetadata for now.